api/v2/routes_account.go

```diff
@@ -143,9 +142,24 @@ func (api *API) registerUserAccount(c *gin.Context) {
 	// log
 	api.l.With("user", forms["username"]).Info("user account registered")
 	// remove hashed password from output
-	userModel.HashedPassword = "scrubbed"
+	user.HashedPassword = "scrubbed"
+	// remove the verification token from output
+	user.EmailVerificationToken = "scrubbed"
+	// format a custom response that includes the user model
+	// and an additional status field
+	type userRegisteredModel struct {
```

Comment:
why not just declare this inline at `gin.H`?

Reply:
good point :man_facepalming:

Reply:
Actually on second thought, I'm not sure this would work? :thinking: `gin.H` expects a `string -> interface`, so the only way I can think of doing this would be:
```golang
userRegisteredModel := gin.H{"user": user, "status": status}
```

This would then break compatibility with the current response expected from this api call?

Reply:
```go
gin.H{"response": gin.H{ "status": status, "user": user } })
```

Reply:
oh i kinda see what you mean - you're adding a new field to user

you can declare an [anonymous struct](https://talks.golang.org/2012/10things.slide#2) inline in the response then

Reply:
like so: https://play.golang.org/p/60n6nR4iBo6

Reply:
shit, thats awesome